cmd/options/outputs.go

```diff
@@ -25,7 +25,26 @@ type Output struct {
 
 func RegisterOutputs(cmd *cobra.Command) *Output {
 	var opts Output
-	cmd.Flags().StringVarP(&opts.Mode, "output", "o", "pretty", "Output mode. Supported modes: "+strings.Join(outputModes, ", "))
+	cmd.Flags().StringVarP(&opts.Mode, "output", "o", "pretty", `
+The output mode controls how Paranoia displays the data, and what data is shown.
+Supported modes are *pretty*, *wide*, *json*, and *pem*.
```

Comment:
Will `StringVarP` render the emphasis (e.g. `*pretty*`)

Reply:
In `--help`, no. In the man page, yes.
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/1119526/197743664-8db8a450-b5c6-4f1f-bb41-3901bc3bf88e.png">
<img width="1512" alt="image" src="https://user-images.githubusercontent.com/1119526/197743829-e1e4daf5-905e-4968-933a-987a225ee6df.png">

Reply:
as long as you're happy with that 😄